stable/ksoc-plugins/README.md

```diff
@@ -47,16 +47,39 @@ ksocGuard:
 
 If admission is blocked, it can be seen in the RAD Security application under the Events tab for the specific cluster. Finally, the plugin also acts as a mutating webhook that simply takes the `AdmissionReview.UID` and adds it as an annotation (`ksoc-guard/admission: xxx`). In the case of a blocked object, this gives RAD Security an identifier to track what would otherwise be an ephemeral event.
 
-### ksoc-runtime plugin
+### ksoc-node-agent plugin
 
-`ksoc-runtime` utilizes system-level probes in order to analyze what is happening at the process level on each node, enabling KSOC to detect in real-time events that may indicate a security breach is occurring. It is not enabled by default. To enable this, please set the following in your values file.
+`ksoc-node-agent` is the plugin responsible for gathering runtime information from the nodes in the cluster. The plugin is not enabled by default. To enable the plugin, set the `enabled` value to `true` in your values file.
 
 ```yaml
-ksocRuntime:
+ksocNodeAgent:
   enabled: true
```

Comment:
Should we take the opportunity to flip on `reachableVulnerabilitiesEnabled` by default?  Happy to defer to a separate update if you want to limit the number of changes here.

Reply:
good point, I'll make it `enabled` by default

Reply:
fixed